apps/charterafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,227 @@
+import linkGroup from "../fields/linkGroup";
+
+const Footer = {
+  slug: "footer",
+  label: {
+    en: "Footer",
+    fr: "Bas de page",
+    pt: "Rodapé",
+  },
+  fields: [
+    {
+      name: "siteDescription",
+      label: {
+        en: "Site Description",
+        fr: "Description du site",
+        pt: "Descrição do Site",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    {
+      name: "projectDescription",
+      label: {
+        en: "Project Description",
+        pt: "Descrição do Projeto",
+        fr: "Description du projet",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    linkGroup(),
```

Comment:
```suggestion
    linkArray(),
```

`linkGroup` creates a link on `group` field. You want link(s) on `array` field.